cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -0,0 +1,125 @@
+{% extends 'content-base.html' %}
+
+{% block title %}
+    Find a Housing Counselor &gt; Consumer Financial Protection Bureau
```

Comment:
Should this be separated by a `|` to be consistent with the Wagtail [base template](https://github.com/cfpb/cfgov-refresh/blob/8c780180d5352fcd4d7f7ee63aee343fc8f91e9f/cfgov/jinja2/v1/_layouts/base-common.html#L53)?

Reply:
Sure!  I'm sure the whole page will get revamped when the front-end rewrite happens, but I may as well be consistent for now.